pkg/api/vlabs/const.go

```diff
@@ -67,3 +71,8 @@ const (
 	// ManagedDisks means that the nodes use managed disks for their os and attached volumes
 	ManagedDisks = "ManagedDisks"
 )
+
+// Network policy
+var (
+	NetworkPolicyValues = [...]string{"", "none", "azure", "calico"}
```

Comment:
I think [...] defines an array, where as [] defines a slice. They are equivalent, except my instincts tell me [...] should be slightly more performant. Anyway, I can convert it to [].